src/containers/FAQ.js

```diff
@@ -1,23 +1,60 @@
 import React from 'react';
 import { FaqBox } from '../components/faqbox';
+import styled from 'styled-components';
+
+const PageWrapper = styled.main`
+  background-color: #E5E5E5;
```

Comment:
Once you merge in master, there is a theme provider for styled components and you can add this color in theme.js and change this to (with the color name replaced):
```CSS
color: ${props=>props.theme.colors.red};
```

Reply:
Sound great! I will merge and add the theme provider